src/web/src/app/app.component.ts

```diff
@@ -134,6 +136,11 @@ export class AppComponent implements OnInit, OnDestroy {
 		this.wspace.isInitialized().subscribe(ok => {
 			this.authorized = ok;
 			this.changeDetectorRef.detectChanges();
+			this.hasWspace = true;
+		});
+
+		this.auth.isAuthorized().subscribe(ok => {
+			this.authorized = ok;
```

Comment:
1. authorized - выставляется в isAuthorized, в wspace.isInitialized не нужен.
2. detectChanges - нужен чтобы обработать изменения свойств которые ты по подписке сделал. Т.е. он должен вызываться после изменения значения свойств.
3.  Блок с auth.isAuthorized лучше поместить перед блока с wspace.isInitialized так как логически это событие наступает раньше.
4. В hasWspace ты должен установить значение которое пришло по подписке, по аналогии с установкой this.authorized в auth.isAuthorized

```suggestion
		this.auth.isAuthorized().subscribe(ok => {
			this.authorized = ok;
			this.changeDetectorRef.detectChanges();
		});

		this.wspace.isInitialized().subscribe(ok => {
			this.hasWspace = ok;
			this.changeDetectorRef.detectChanges();
		});
```

Reply:
Готово.